project/Build.scala

```diff
@@ -804,7 +810,17 @@ object Build {
       previousArtifactSetting,
       mimaBinaryIssueFilters ++= BinaryIncompatibilities.SbtPlugin,
 
-      addSbtPlugin("org.scala-native" % "sbt-crossproject" % "0.2.1"),
+      /* This works around a bug in ^^ from sbt (should be just addSbtPlugin).
+       * We inline the definition of addSbtPlugin and fix the sbt binary version.
+       */
+      libraryDependencies += {
+        val sbtV =
+          if ((sbtVersion in pluginCrossBuild).value.startsWith("1.0.")) "1.0"
+          else (sbtBinaryVersion in update).value
+        val scalaV = (scalaBinaryVersion in update).value
+        Defaults.sbtPluginExtra(
+            "org.scala-native" % "sbt-crossproject" % "0.2.1", sbtV, scalaV)
+      },
```

Comment:
I had to do this to resolve the build failure. It seems `^^` does not correctly patch whatever's necessary for a dependency from an sbt plugin to another sbt plugin to work.

I'll report the bug later.